scripts/lottery/process_lottery_data.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Function to display usage instructions
+usage() {
+    echo "Usage: $0 [environment] <drupal-root> <site-uri> <item-limit>"
+    echo "Environment should be 'lando' or 'server'. If not provided, defaults to 'server'."
+    exit 1
+}
+
+# Check if the correct number of arguments is provided and specify which parameter is missing
+if [ -z "$1" ]; then
+    echo "Error: Missing drupal-root parameter."
+    usage
+elif [ -z "$2" ]; then
+    echo "Error: Missing site-uri parameter."
+    usage
+elif [ -z "$3" ]; then
+    echo "Error: Missing item-limit parameter."
+    usage
+fi
+
+# Determine if the environment parameter is provided
+if [ "$#" -eq 3 ]; then
+    ENVIRONMENT="server"
+    ROOT="$1"
+    URI="$2"
+    ITEM_LIMIT="$3"
+else
+    ENVIRONMENT="$1"
+    ROOT="$2"
+    URI="$3"
+    ITEM_LIMIT="$4"
+fi
+
+# Set Drush path based on environment
+if [ "$ENVIRONMENT" = "lando" ]; then
+    DRUSH_PATH="/app/vendor/bin/drush"
+elif [ "$ENVIRONMENT" = "server" ]; then
+    DRUSH_PATH="/usr/bin/drush10"
```

Comment:
Can't we make it just `usr/bin/drush` instead of `/usr/bin/drush10`? Not every server will have drush10 as their alias.

Reply:
On dev server there is no `usr/bin/drush` directory, we are using `usr/bin/drush10`

Reply:
But we will have to make sure that if this alias is not present then we should create one.

Reply:
Added fallback path as "usr/bin/drush"